pkg/bootstrap/docker/up.go

```diff
@@ -130,6 +132,7 @@ func NewCmdUp(name, fullName string, f *osclientcmd.Factory, out io.Writer) *cob
 	cmd.Flags().IntVar(&config.ServerLogLevel, "server-loglevel", 0, "Log level for OpenShift server")
 	cmd.Flags().StringSliceVarP(&config.Environment, "env", "e", config.Environment, "Specify key value pairs of environment variables to set on OpenShift container")
 	cmd.Flags().BoolVar(&config.ShouldInstallMetrics, "metrics", false, "Install metrics (experimental)")
+	cmd.Flags().BoolVar(&config.AlwaysPull, "always-pull", false, "Always pull the latest images")
```

Comment:
how do i tell it to use my local images (which may not match to "config.ImageVersion")?  specify version as "latest" and always-pull to false?